docs/CONTRIBUTING.md

```diff
@@ -2,7 +2,15 @@
 
 ## Requirements
 
-Coder requires Go 1.18+, Node 14+, and GNU Make.
+You'll need the following tools:
+- Go 1.18+
+- Node 14+
+- GNU Make
+- [`shfmt`](https://github.com/mvdan/sh#shfmt)
+- [`nfpm`](https://nfpm.goreleaser.com/install/)
+- [`zstd`](https://github.com/horta/zstd.install)
+- [`pg_dump`]
+  - on macOS, run `brew install libpq zstd`
```

Comment:
I wasn't sure if this would confuse people since I technically list `zstd` twice, but also because I put `pg_dump` but then tell people to install `libpq` and `zstd`. Open to better ideas!

Reply:
We could break out into separate Linux and MacOS sections, perhaps?

Reply:
Good idea! I went with this for now: https://github.com/coder/coder/pull/3541/commits/6aa9e835a93e1c5d61cf655ddeda4699582875a1